portfolio/src/main/webapp/my-travels/travels.js

```diff
@@ -1,29 +1,65 @@
 const placesCoords = {
-                    "berlin" : {lat: 52.521999, lng: 13.413016},
-                    "default" : {lat: 52.048947, lng: 20.442758},
-                    "disneyland" : {lat: 48.869486, lng: 2.780874},
-                    "krakow" : {lat: 50.061683, lng: 19.937111},
-                    "kolobrzeg" : {lat: 54.186629, lng: 15.570729},
-                    "matterhorn" : {lat: 45.938319, lng: 7.731824},
-                    "zurich" : {lat: 47.349534, lng: 8.491872}
-                   }
+    'berlin' : {lat: 52.521999, lng: 13.413016},
+    'default' : {lat: 52.048947, lng: 20.442758},
+    'disneyland' : {lat: 48.869486, lng: 2.780874},
+    'krakow' : {lat: 50.061683, lng: 19.937111},
+    'kolobrzeg' : {lat: 54.186629, lng: 15.570729},
+    'matterhorn' : {lat: 45.938319, lng: 7.731824},
+    'zurich' : {lat: 47.349534, lng: 8.491872}
+    }
+
+const placesDescriptions = {
```

Comment:
you could merge this dictionary with `placesCoords` like

```javascript
const places = {
  berlin : {
    coords: {...},
    description: '...'
  }
}
```

Side note: what about storing places data in a different file (e.g. as JSON) and retrieve it from the server?

Reply:
I was thinking about moving it to backend, but I wasn't not sure if it is time efficient to send requests (my comments are fetching really long and there aren't many of them). But I'll try to make it work :)

Reply:
fair point, however I think it is a nice learning experience and it allows changing data independent from the code itself.

Reply:
Exactly, I'm working on it now. Regarding capstone project (since we have to store a lot of data) I'm thinking about doing some experiments on this small example to see which data storage may suit our big project best :)

Reply:
**Update**: I've read about best way to store not changing data and general way is to make the static content, then load it to JS. [Explanation from stack](https://stackoverflow.com/questions/34461314/serve-content-from-file-vs-database-in-node)
It has all advantages of retrieving it from server -- you don't store hardcoded data, there is parsing JSON part. Also it's less energy consuming -- https://blog.stackpath.com/static-content/ and browser will cache it.

Reply:
It is totally up to you as it is a learning experience. In a real application, storing a JSON file might be most efficient, depending on how often that data changes i.e. you travel :-)

Also keep in mind that you wanted to work on another API, have this weeks tutorial, and  work on the capstone project design doc. Don't overwork yourself.